index.html

```diff
@@ -12704,13 +12705,14 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>When using <code>aria-roledescription</code>, authors SHOULD also ensure that:</p>
 				<ol>
 					<li>The element to which <code>aria-roledescription</code> is applied has a valid <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role or has an implicit WAI-ARIA role semantic.</li>
-					<li>The value of <code>aria-roledescription</code> is not empty or does not contain only whitespace characters.</li>
+					<li>The value of <code>aria-roledescription</code> is not empty or does not contain only [=ascii whitespace|whitespace=] characters.</li>
 				</ol>
 				<p class="note">Depending on the assistive technology, user verbosity settings, or other factors, certain elements' role descriptions might not be conveyed. If specifying <code>aria-roledescription</code> on such elements, then the custom role descriptions may also not be conveyed by these assistive technologies.</p>
+				<p>Additionally, authors MUST NOT specify <code>aria-roledescription</code> on an element which has an explicit or implicit WAI-ARIA role where <code>aria-roledescription</code> is <a href="#prohibitedattributes">prohibited</a>.</p>
 				<p>User agents MUST NOT expose the <code>aria-roledescription</code> property if any of the following conditions exist:</p>
 				<ol>
 					<li>The element to which <code>aria-roledescription</code> is applied has an explicit or implicit WAI-ARIA role where <code>aria-roledescription</code> is <a href="#prohibitedattributes">prohibited</a>.</li>
-					<li>The value of <code>aria-roledescription</code> is empty or contains only whitespace characters.</li>
+					<li>The value of <code>aria-roledescription</code> is the empty string.</li>
```

Comment:
is undefined the same as the empty string? We might want to say do not expose if undefined as well.

Reply:
but it didn't say that before, and this change is only about removing white space, so maybe that is not necessary.

Reply:
```suggestion
					<li>The value of <code>aria-roledescription</code> is undefined or the empty string.</li>
```